components/infobox/commons/infobox_person.lua

```diff
@@ -0,0 +1,280 @@
+local Class = require('Module:Class')
+local Cell = require('Module:Infobox/Cell')
+local BasicInfobox = require('Module:Infobox/Basic')
+local Template = require('Module:Template')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+local Namespace = require('Module:Namespace')
+local Links = require('Module:Links')
+local Localisation = require('Module:Localisation').getLocalisation
+local Flags = require('Module:Flags')
+--local GetBirthAndDeath = require('Module:???')._get
+
+--the following 3 lines as a temp workaround until the Birth&Death stuff is implemented:
+local function GetBirthAndDeath()
+	return '', nil, nil, nil
+end
+
+local Player = Class.new(BasicInfobox)
+
+local Language = mw.language.new('en')
+local _LINK_VARIANT = 'player'
+local shouldStoreData
+
+function Player.run(frame)
+	local player = Player(frame)
+	return player:createInfobox(frame)
+end
+
+function Player:createInfobox(frame)
+	local infobox = self.infobox
+	local args = self.args
+	shouldStoreData = Player:shouldStoreData(args)
+
+	local earnings = self:calculateEarnings(args)
+	Variables.varDefine('earnings', earnings)
+	if earnings == 0 then
+		earnings = nil
+	else
+		earnings = '$' .. Language:formatNum(earnings)
+	end
+	local nameDisplay = self:nameDisplay(args)
+	local role = self:getRole(args)
+	local birthDisplay, deathDisplay, birthday, deathday
+		= GetBirthAndDeath(
+			args.birth_date,
+			args.birth_location,
+			args.death_date,
+			role.category,
+			shouldStoreData
+		)
+	local status = self:getStatus(args)
+
+	infobox:name(nameDisplay)
+	infobox:image(args.image, args.defaultImage)
+	infobox:centeredCell(args.caption)
+	infobox:header(self:getInformationType(args) .. ' Information', true)
+	infobox:cell('Name', args.name)
+	infobox:cell('Romanized Name', args.romanized_name)
+	infobox:cell('Birth', birthDisplay)
+	infobox:cell('Died', deathDisplay)
+	infobox:cell('Status', status.display)
+	infobox:cell(role.title or 'Role', role.display)
+	infobox:fcell(Cell :new('Country')
+						:options({})
+						:content(unpack(self:_createLocations(args, role.category)))
+						:make()
+			)
```

Comment:
```suggestion
	infobox:fcell(Cell	:new('Country')
						:options({})
						:content(unpack(self:_createLocations(args, role.category)))
						:make()
			)
```

Weird that Github doesn't handle this better, looks fine aside from this in VS Code